libcheckout

```diff
@@ -118,6 +119,14 @@ function checkout::cleanupcache {
 
 }
 
+function checkout::checkrev {
+  git rev-list HEAD..$SEMAPHORE_GIT_SHA 2>/dev/null
+  if [ $? -ne 0 ]; then
+    echo "Rev: ${SEMAPHORE_GIT_SHA} not found"
```

Comment:
Nitpick from me :) Please change `Rev` to `Revision`.

Reply:
Sorted 👍